installers/custom_ipxe/main.go

```diff
@@ -45,6 +52,11 @@ func bootScript(ctx context.Context, j job.Job, s *ipxe.Script) {
 		return
 	}
 
+	if i.extraIPXEVars != nil {
```

Comment:
I don't this this check is necessary. I believe an empty array/nil value here will have nothing to iterate and will just move on. [example](https://goplay.tools/snippet/VOVP-oixwnI).

Reply:
Good catch, thank you!